src/System.Private.Uri/tests/FunctionalTests/UriIsWellFormedUriStringTest.cs

```diff
@@ -468,5 +468,20 @@ public static void TestIsWellFormedUriString(string uriString, bool expected)
         {
             Assert.Equal(expected, Uri.IsWellFormedUriString(uriString, UriKind.RelativeOrAbsolute));
         }
+
+        public static IEnumerable<object[]> UriIsWellFormedUnwiseStringData =>
+        new List<object[]>
+        {
+            new object[] { "https://www.contoso.com/?a=%7B%7C%7D&b=%E2%80%99" },
+            new object[] { "https://www.contoso.com/?a=%7B%7C%7D%E2%80%99" }
+        };
+
+        [Theory]
+        [MemberData(nameof(UriIsWellFormedUnwiseStringData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public void UriIsWellFormed_AbsoluteUnicodeWithUnwise_Success(string uriString)
```

Comment:
Can you also add some negative tests to ensure that unescaped `{|}` characters return _false_?